src/wizard/controller/frmResultSummaryPanel.py

```diff
@@ -242,9 +242,9 @@ def createResult(self):
                                 YLocationUnitsID=yUnit,
                                 ZLocation=z,
                                 ZLocationUnitsID=zUnit,
-                                SpatialReferenceID=sr,
-                                IntendedTimeSpacing=timeSpacing,
-                                IntendedTimeSpacingUnitsID=timeUnit)
+                                SpatialReferenceID=sr)
+                                # IntendedTimeSpacing=timeSpacing,
```

Comment:
These still need to be set(intendedTimeSpacing and IntendedTimeSpacingUnitsID), we just need to get the values from the main setup page. the other time spacing location.